.ruby-version

```diff
@@ -1 +1 @@
-3.3.1
+3.3.4
```

Comment:
Is there a Rubocop parser we can update too? It prints a little warning when the patch version is different